### PR TITLE
as.data.frame method

### LINKED ISSUE #61: as.data.frame method
**Is your feature request related to a problem? Please describe.**
No, not a problem. Might be nice to have a shortcut to each dataframe to the global environment

**Describe the solution you'd like**
as.data.frame(sT_object)

**Describe alternatives you've considered**
None

### PR DESCRIPTION
**Pull request by aghaynes**
closes #61

### CODE REVIEW

File: R/asdataframe.R [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+#'                                package = "secuTrialR")
+#' # load all export data
+#' sT_export <- read_secuTrial_export(data_dir = export_location)
+#' # add files to global environment
+#' as.data.frame(sT_export)
+#' # add files to global environment, removing the project name from the file names
+#' as.data.frame(sT_export, regex = "ctu05")
+as.data.frame.secuTrialdata <- function(object,
+                                        data.frames = NULL,
+                                        meta = TRUE,
```
Comment (PatrickRWright) [inline R/asdataframe.R]: Naively I was thinking `meta = FALSE` would be better. What is your reason to set the default to `TRUE`?

File: R/asdataframe.R [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+#' export_location <- system.file("extdata",
+#'                                "s_export_CSV-xls_CTU05_longnames_sep_ref.zip",
+#'                                package = "secuTrialR")
+#' # load all export data
+#' sT_export <- read_secuTrial_export(data_dir = export_location)
+#' # add files to global environment
+#' as.data.frame(sT_export)
+#' # add files to global environment, removing the project name from the file names
+#' as.data.frame(sT_export, regex = "ctu05")
+as.data.frame.secuTrialdata <- function(object,
```
Comment (PatrickRWright) [inline R/asdataframe.R]: Technically I would prefer it if you called it `as_data_frame` in accordance with the other functions in the package. Would you mind?

File: R/asdataframe.R (hunk 1 above)
Comment (aghaynes) [reply R/asdataframe.R]: Thats a fair point. No particular reason at all actually...

File: R/asdataframe.R (hunk 2 above)
Comment (aghaynes) [reply R/asdataframe.R]: I prefer it this way which is in line with the R norm... 
image

Comment (PatrickRWright) [reply R/asdataframe.R]: Ok, makes sense.

**Comment by PatrickRWright** [comment]
From my side its fine. Can I merge it?

**Comment by aghaynes** [comment]
it's finished as far as I'm concerned

**Pull request merged**